src/vs/workbench/contrib/tasks/browser/taskTerminalStatus.ts

```diff
@@ -107,6 +100,7 @@ export class TaskTerminalStatus extends Disposable {
 		} else if (terminalData.problemMatcher.maxMarkerSeverity === MarkerSeverity.Info) {
 			terminalData.terminal.statusList.add(INFO_TASK_STATUS);
 		}
+		this._audioCueService.playAudioCue(event.exitCode && event.exitCode !== 0 ? AudioCue.taskFailed : AudioCue.taskCompleted);
```

Comment:
This would be fine with `!==` (not `!=`)

```suggestion
		this._audioCueService.playAudioCue(event.exitCode !== 0 ? AudioCue.taskFailed : AudioCue.taskCompleted);
```

Reply:
there might not be an exit code - like when a background watch task completes